### PR TITLE
added image and updated readme

### PR DESCRIPTION
**Pull request by chrisosbii**
looks good

### CODE REVIEW

**Comment by chrisosbii** [comment]
hmm how do you approve?

File: README.md [hunk 1]
```diff
@@ -1,2 +1,3 @@
 # This is the Ultra Cool Repo on Duck Turtles
 
+![Duck Turtle From Avatar the Last Airbender](./assets/images/duckturtle.png)
```
Comment (DuckTurtle) [inline README.md:3]: aprove

**Review by DuckTurtle — COMMENTED** [review]
aprove

File: README.md (hunk 1 above)
Comment (DuckTurtle) [reply README.md:3]: hit add your review next to review required

**Comment by chrisosbii** [comment]
Shouldnt there be an approve button?

**Comment by DuckTurtle** [comment]
owner aproved

File: README.md (hunk 1 above)
Comment (DuckTurtle) [inline README.md:3]: apove

**Pull request merged**